### ISSUE TITLE
Acceptance tests for Ubuntu timeout

### ISSUE THREAD

**Issue by jarretlavallee** [puppetlabs/puppetlabs-pe_metrics_dashboard#22]
During the travis acceptance tests on Ubuntu 16.04, there is a timeout when running an apt-update. It looks like this issue is only seen after adding the grafana repo, and can be an issue with the upstream repo. 

~~~
  Info: /Stage[main]/Pe_metrics_dashboard::Repos/Apt::Source[grafana]/Apt::Setting[list-grafana]/File[/etc/apt/sources.list.d/grafana.list]: Scheduling refresh of Class[Apt::Update]
  Info: Class[Apt::Update]: Scheduling refresh of Exec[apt_update]
  Error: /Stage[main]/Apt::Update/Exec[apt_update]: Failed to call refresh: Command exceeded timeout
  Error: /Stage[main]/Apt::Update/Exec[apt_update]: Command exceeded timeout
~~~

**Comment by jarretlavallee** [comment]
It turns out that this is because the influxdb.list created for apt uses https and the `apt-transport-https` package was not installed in the docker image. Adding this to the node list resolves the issue.

**Comment by jarretlavallee** [comment]
resolved by #37
